live/src/main/scala/org/dbpedia/extraction/live/LiveExtractionConfigLoader.scala

```diff
@@ -90,7 +90,7 @@ object LiveExtractionConfigLoader
     val lang = Language.apply(landCode)
     val articlesSource : Source =
       if (item.getXML.isEmpty)
-        WikiSource.fromPageIDs(List(item.getItemID), new URL(apiURL), lang)
+        WikiSource.fromRevisionIDs(List(item.getItemID), new URL(apiURL), lang)
```

Comment:
can we change that back? the other feeders (mappings / unmodified) need the page id